client-app/shared/static-content/components/title-block.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="pt-6 pb-16 lg:py-24">
+    <div class="w-full max-w-screen-2xl mx-auto px-5 md:px-12">
+      <h2 v-if="model.heading == 'h2'" class="text-2xl">{{ model.title }}</h2>
+      <h3 v-if="model.heading == 'h3'" class="text-2xl">{{ model.title }}</h3>
```

Comment:
can be on one line:
`<component :is="model.heading" class="text-2xl">{{ model.title }}</component>`

Reply:
fixed